blogService/src/main/java/blogService/service/BlogService.java

```diff
@@ -0,0 +1,25 @@
+package blogService.service;
+
+import blogService.dto.BlogDto;
+import blogService.entity.Blog;
+import blogService.mapper.BlogMapper;
+import blogService.repository.BlogRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BlogService {
+    private final BlogRepository blogRepository;
+    private final BlogMapper blogMapper;
+
+    public BlogService (final BlogRepository blogRepository,
+                        final BlogMapper blogMapper) {
+        this.blogRepository = blogRepository;
+        this.blogMapper = blogMapper;
+    }
+
+    public Long createBlog(BlogDto blogDto) {
```

Comment:
dobrą praktyką jest aby argumenty były oznaczone jako final (przyjmijmy to teraz za standard - możesz poprawić pozostałe miejsca - również w kontrolerach)

Reply:
ok